packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/LayoutUsdArs/index.tsx

```diff
@@ -0,0 +1,255 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import Currencies from '@core/exchange/currencies'
+import { AgentType } from '@core/types'
+import { Icon, Text } from 'blockchain-info-components'
+import { DisplayIcon, DisplaySubTitle, DisplayTitle } from 'components/BuySell'
+import CopyClipboardButton from 'components/Clipboard/CopyClipboardButton'
+import { ExpansionPanel } from 'components/ExpansionPanel'
+import { FlyoutWrapper, Row, Title, Value } from 'components/Flyout'
+import { BankDWStepType } from 'data/types'
+
+import { Props as OwnProps, SuccessStateType } from '../..'
+
+const RowCopy = styled(Row)`
+  display: flex;
```

Comment:
You could use Flex component from component package. You will find examples like
`<Flex flexDirection='row' justifyContent='space-between'>`